test/llvm_ir_correct/tbaa_multimod_01_caller.f90

```diff
@@ -0,0 +1,18 @@
+! This test contians two files, tbaa_multimod_01_caller.f90 and tbaa_multimod_01_callee.f90
+! RUN: %flang -Wl,-mllvm,-aa-trace -fuse-ld=lld -flto=full -O3 %s %S/tbaa_multimod_01_callee.f90 -o - 2>&1 1>- | FileCheck %s
```

Comment:
These tests are failing in the CI jobs. I understand that `-aa-trace` only works whne LLVM has assertions enabled, but I believe that the GitHub workflow for classic-flang-llvm-project already configures LLVM with `-DLLVM_ENABLE_ASSERTIONS=ON`:

https://github.com/flang-compiler/classic-flang-llvm-project/blob/release_17x/.github/workflows/pre-compile_llvm.yml#L65

https://github.com/flang-compiler/classic-flang-llvm-project/blob/release_17x/.github/workflows/UbuntuDockerfile#L54

@1997alireza  Can you build classic-flang-llvm-project using the same commands as used by the CI jobs, and confirm whether you can reproduce the test failures in your environment?

Reply:
I think I know what the problem is. Your tests specify `-fuse-ld=lld` but `lld` is not available in the CI. And for some reason, the errors from stderr are eaten by `FileCheck`, and not shown in the `llvm-lit` output, causing confusion for us.